Paintroid/src/androidTest/java/org/catrobat/paintroid/test/utils/RecyclerViewMatcher.kt

```diff
@@ -0,0 +1,58 @@
+package org.catrobat.paintroid.test.utils
+
+import android.content.res.Resources
+import android.content.res.Resources.NotFoundException
+import android.view.View
+import androidx.recyclerview.widget.RecyclerView
+import org.hamcrest.Description
+import org.hamcrest.Matcher
+import org.hamcrest.TypeSafeMatcher
+
+class RecyclerViewMatcher(private val recyclerViewId: Int) {
+
+    fun atPositionOnView(position: Int, targetViewId: Int): Matcher<View> {
+        return object : TypeSafeMatcher<View>() {
+            var resources: Resources? = null
+            var childView: View? = null
+
+            @SuppressWarnings("SwallowedException")
```

Comment:
?
https://media.giphy.com/media/4cuyucPeVWbNS/giphy.gif

Reply:
This has to be done, as Kotlin will otherwise always complain, but in this case this has to be added, as described above we can not re-throw it nicely